README.md

```diff
@@ -184,6 +185,7 @@ Or you can try to do this by hooking into protractor functions, to ensure that a
 
 We recommend to do it in the page objects, because that is usually the place where you know, that something worthy of recording as a step happened.
 
+If you are using Typescript (old Jasmine syntax or the new Fluent DSL), you can follow the example [exampleBasicJasmineWithTypeScript.spec.ts](example/test/exampleBasicJasmineWithTypeScript.spec.ts) and the used page object [startPage.ts](example/test/pages/startPage.ts) as a reference on how to use a [Typescript Decorator](http://www.typescriptlang.org/docs/handbook/decorators.html) to perform this in an easy manner.
```

Comment:
I would prefer to integrate that example into our full Fluent DSL example of how we recommend to write tests, such that we have that example in there as well.